system/extensions/esp32/wifi.toit

```diff
@@ -118,8 +118,8 @@ class WifiServiceDefinition extends NetworkServiceDefinitionBase:
   address resource/NetworkResource -> ByteArray:
     return (state_.module as WifiModule).address.to_byte_array
 
-  rssi resource/NetworkResource -> int?:
-    return (state_.module as WifiModule).rssi
+  get_ap_info resource/NetworkResource -> List:
+    return (state_.module as WifiModule).get_ap_info
```

Comment:
Is this where the exception is thrown if we're not connected? Is it a nice and understandable exception?

```suggestion
  ap_info resource/NetworkResource -> List:
    return (state_.module as WifiModule).ap_info
```